### DISCUSSION TITLE
Citra mmj custom layout

### DISCUSSION THREAD

**Discussion by GevOLA31** [weihuoya/citra#958] [Q&A]
Hi everybody,
I have 2 questions about the custom layout, I can't find answers on the web, so here I am, asking for your help :)
I would like to reset my custom layout setting, cause I don't know how, the main screen corners handles are stuck, and I can't move the main screen, just the secondary screen is anymore movable and re-sizable.
Is there a way to reset the custom layout ?
In addition, I wonder to set the custom layout to fit pixel perfectly to my screen, so, instead of using touchscreen, wich is really not accurate, is there a way to set coordinates of the corners manually in a configuration file ?
Thanks in advance for your support <3

**Comment by GevOLA31** [comment]
While still searching for answers, I've found this webtool wich propose to generate a custom layout :
[https://jesuscc1993.github.io/miscellaneous/citra-layout-generator/](url)
That's for citra's main project, not mmj, but I think it should be very close or maybe the same for mmj.
For the moment, I didn't find the configuration file location on my android device where I could put the code generated by this webtool.

So, new question, where can I find the mmj config file on android ?

**Reply by android-1995** [reply]
config/input-layout.ini.
normal:citra-emu/config/input-layout.ini
Storage Access:Android/data/org.citra.emu/files/citra-emu/config/input-layout.ini
